### PR TITLE
Fix pushing plugins json file to `gh-pages`

### PR DESCRIPTION
**Pull request by AhmedBasem20**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -106,6 +105,9 @@ jobs:
         npm run build
       working-directory: ./aiida-registry-app
 
+    - name: Add plugins file to the build folder
+      run: cp aiida-registry-app/src/plugins_metadata.json ./aiida-registry-app/dist/
```
Comment (unkcpz) [inline .github/workflows/ci.yml:109]: I think I understand why previously the file disappeared. It was wiped by the npm build in the `dist` folder. 
Therefore, I think it is okay here to make the following change:

```suggestion
      run: cp plugins_metadata.json ./aiida-registry-app/dist/
```

Comment (unkcpz) [reply .github/workflows/ci.yml:109]: After this change, the "Move JSON file to the React project" step in `generate_metadata` action can be removed. Can you give it a try?

Comment (AhmedBasem20) [reply .github/workflows/ci.yml]: Done, but removing the move step causes an error as the file cannot be accessed.
https://github.com/aiidateam/aiida-registry/actions/runs/6310303316/job/17131993630

Comment (unkcpz) [reply .github/workflows/ci.yml]: I see, then everything makes sense. I remember you said before that you are not very sure why need to put the file in both `src` and `dist`. Now we know they are all necessary for building and for attaching to dist on gh_page branch. 
Thus, I think your initial implementation of this PR is 100% correct. 

let's wait preview test to pass and we can merge it.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -106,6 +105,9 @@ jobs:
         npm run build
       working-directory: ./aiida-registry-app
 
+    - name: Add plugins file to the build folder
+      run: cp plugins_metadata.json ./aiida-registry-app/dist/
```
Comment (unkcpz) [inline .github/workflows/ci.yml:109]:
```suggestion
      run: cp plugins_metadata.json aiida-registry-app/dist/
```

File: .github/workflows/webpage.yml [hunk 3]
```diff
@@ -32,21 +32,18 @@ jobs:
...
         node-version: '18.x'
     - name: Install npm dependencies and build
       run: |
-        echo $VITE_PR_PREVIEW_PATH
         npm install
         npm run build
       working-directory: ./aiida-registry-app
 
+    - name: Add plugins file to the build folder
+      run: cp aiida-registry-app/src/plugins_metadata.json ./aiida-registry-app/dist/
```
Comment (unkcpz) [inline .github/workflows/webpage.yml:45]:
```suggestion
      run: cp aiida-registry-app/src/plugins_metadata.json aiida-registry-app/dist/
```

**Review by unkcpz — CHANGES_REQUESTED** [review]
Nitpick to align the format with `run: cp plugins_metadata.json aiida-registry-app/src/`

**Review by unkcpz — APPROVED** [review]
Great, thanks!

**Pull request merged**